packages/atlas/src/components/_overlays/Popover/Popover.tsx

```diff
@@ -106,8 +146,8 @@ const TriggerContainer = styled.div`
   height: max-content;
 `
 
-const ContentContainer = styled.div`
-  transition: 150ms cubic-bezier(0.25, 0.01, 0.25, 1);
+const ContentContainer = styled.div<{ animation?: boolean }>`
+  transition: ${({ animation }) => (animation ? ' 150ms cubic-bezier(0.25, 0.01, 0.25, 1)' : 'unset')};
```

Comment:
@drillprop @toiletgranny Do you think guys we could define it as design token?

Reply:
I think we all need some context here, so let me try to explain. Rafał is referring to the animation of the Popover component(which is mostly used for the context menu) and it has some old value  - `150ms cubic-bezier(0.25, 0.01, 0.25, 1)'`. In this PR I only add a possibility to  **disable** animation for the purpose of using settings popover. So this value is just a default animation transition when it's not **disabled**
@toiletgranny we could change this to our token or we could add another one if you want. But on the other hand, I don't want to increase the scope of this PR any further.

Reply:
@drillprop is right, our intention was not to change the animation here, it was to have an option to disable it. However, since we're here, and in case we won't be visiting this component in a very long while, I guess @rafalpawlow has a good point that it wouldn't hurt to change the hardcoded animation value with our token. The closest one matching these settings would be `var(--animation-transition-fast)`. Would that be a simple thing to do, @drillprop?